src/translations/en/common.json

```diff
@@ -48,6 +48,7 @@
   "title": "Title",
   "type": "Type",
   "users": "Users",
+  "will_be_available": "Will be available",
```

Comment:
kunne godt brukt kontekst og `Will be available {{ date }}`, men det er ikke så nøye